### DISCUSSION TITLE
picture background for pinned discussions

### DISCUSSION THREAD

**Discussion by Ckath** [community/community#2896] [Discussions] [answered]
currently its only possible to set a pattern and gradient when making a 'pinned discussion'
![image](https://user-images.githubusercontent.com/15038414/101629157-d0b3c300-3a18-11eb-9936-1f6228fb39ab.png)

I'd prefer if instead, or aditionally, we could also set a picture background here. I feel like it'd really help keep projects within the company look and feel.

**Comment by becca** [comment, marked as answer]
@Ckath early on we heard from communities about this request but most didn't have enough high quality images to upload that would fit the pinned discussions specs. 

some questions i'd love to gain some clarity on a bit more:
* do you expect to change the pinned discussions often or will they be pretty static?
* apprx how many images (within pinned discussions) would you expect to have highlighted on your discussions page at any given time?
* what kind of images are you hoping to use here? are they logos? images that are present within the discussion post itself?

we can definitely note this as feedback here and see if others request this again with Discussions being more widely available.

**Reply by Ckath** [reply]
Just a quick response from at least my usecases, not sure what others would
use it for. It just mostly seemed odd to me there was customisation to the point of being able to pick a gradient and pattern but then not the full freedom to just put an image.
> * do you expect to change the pinned discussions often or will they be pretty static?

Most likely static
> * apprx how many images (within pinned discussions) would you expect to have highlighted on your discussions page at any given time?

Probably only a couple, wouldn't expect a ton to be pinned to begin with.
> * what kind of images are you hoping to use here? are they logos? images that are present within the discussion post itself?

most likely either a logo for generic topics such as ideabox etc, or a quick screenshot of the problem/topic at hand.

**Reply by megancoghlan-vertex** [reply]
I echo the request - We'd like to use an image to maintain brand consistency.

**Reply by etpalmer63** [reply]
Is it possible to do this now? Or is this still just a feature request?

**Comment by annaesvensson** [comment]
I like how the new pinned discussions look:

![Screen Shot 2023-05-06 at 14 32 45](https://user-images.githubusercontent.com/14218799/236624513-aeaab07f-2058-488b-a579-f5479ea5be83.png)

If possible I would like to chose more background colours, full rainbow please. 🌈

**Comment by BrajBliss** [comment]
I can't seem to edit the pattern anymore. I can only change the color from pinned discussion.
